### PR TITLE
Design fix

### PR DESCRIPTION
**Pull request by qaseem-aslam-brainx**

### CODE REVIEW

File: sponser_dashboard.html [hunk 1]
```diff
@@ -4,194 +4,9 @@
...
-            padding: 5px 20px 5px 20px;
-        }
-        .options{
-            border: 1px solid #9e9898;
-            border-radius: 8px;
-            text-decoration: none;
-            padding: 5px 10px 5px 10px;
-        }
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="sponsor-dashboard-style.css">
```
Comment (sobanakram) [inline sponser_dashboard.html]: We don't need to add `style` in file name,
better way is to create folder for css

File: sponser_dashboard.html [hunk 2]
```diff
@@ -203,97 +18,92 @@
...
         <nav>
-            <div style="position: absolute;right: 350px; top: 12px;">
-                <div class="circleoutline">
-                    <button onclick="showLogout()" class="circlebutton" type="button">J</button>
-                    <div id="logout" onclick="window.location.href = 'login.html';">
-                        <i style="color: #C4183C;float: left;padding: 13px 0px 0px 0px;margin-left: 10px;" class="fas fa-sign-out-alt"></i>
-                        <span id="sign-out">Sign Out</span><br>
-                        <p style="font-size: 14px; margin-left: 13px;">john.doe@brainxtech.com</p>
-                    </div>
+            <div class="circleoutline">
```
Comment (sobanakram) [inline sponser_dashboard.html]: use `-` for classes names.
and outline is different then border.

File: sponser_dashboard.html [hunk 3]
```diff
@@ -203,97 +18,92 @@
...
+                <button onclick="showLogout()" class="circlebutton" type="button">J</button>
+                <div id="logout" onclick="window.location.href = 'login.html';">
+                    <i class="fas fa-sign-out-alt icon-style"></i>
+                    <span id="sign-out">Sign Out</span><br>
```
Comment (sobanakram) [inline sponser_dashboard.html]: using style in css class name is not necessary.

File: sponser_dashboard.html [hunk 4]
```diff
@@ -203,97 +18,92 @@
...
         </nav>
-        <div style="margin-top: 45px;">
-            <h1 id="fair-heading">Virtual Fair</h1>
-        </div>
-        <div>
-            <div class="column1" style="margin-left: 250px;">
-                <ul>
+        <div class="container">
+            <div >
+                <h1 id="fair-heading">Virtual Fair</h1>
```
Comment (sobanakram) [inline sponser_dashboard.html]: avoid adding `id` if we don't need it.

File: sponser_dashboard.html [hunk 5]
```diff
@@ -203,97 +18,92 @@
...
+            </div>
+            
+            <div class="column1">
+                <ul class="fair-tabs">
                     <li><a href="#">Event Detail</a></li>
                     <li><a href="#">Vendor</a></li>
-                    <li><a href="#" style="border-bottom: 3px solid blue;color: blue">Sponser</a></li>
+                    <li><a href="#" class="sponsor-style">Sponser</a></li>
```
Comment (sobanakram) [inline sponser_dashboard.html]: separate class is used to show active tab.
`active` adding this class should make active and change styling

File: sponser_dashboard.html [hunk 6]
```diff
@@ -203,97 +18,92 @@
...
+                            <th>#</th>
+                            <th>Sponsor Logo</th>
+                            <th>Sponsor Name</th>
+                            <th>Sponsor Website</th>
+                            <th>Status</th>
+                            <th>Action</th>
+                        </tr>
+                        <tr>
+                            <td>1</td>
+                            <td><img style="margin-top: 5px;" src="logo.png" ></td>
```
Comment (sobanakram) [inline sponser_dashboard.html]: Remove inline styling.

File: sponser_dashboard.html [hunk 7]
```diff
@@ -203,97 +18,92 @@
...
+                            <td>Virtual Fair</td>
+                            <td>www.virtualfair.com</td>
+                            <td><span class="status-active">Active</span></td>
```
Comment (sobanakram) [inline sponser_dashboard.html]: Use two separate classes `status` `active`.

File: sponsor-dashboard-style.css [hunk 8]
```diff
@@ -0,0 +1,165 @@
+#fair-heading{
```
Comment (sobanakram) [inline sponsor-dashboard-style.css]: Avoid styling based in `id`

File: sponsor-dashboard-style.css [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+ul.fair-tabs li {
+    float: left;
+    font-size: 16px;
+    font-family: Avenir;
+}
+
+ul.fair-tabs li a{
+    display: flex;
+    color: black;
+    padding: 10px 10px 10px 10px;
```
Comment (sobanakram) [inline sponsor-dashboard-style.css]: we can use shorthand version like `padding: 10px;`

File: sponsor-dashboard-style.css [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+    font-family: Avenir;
+}
+ul.fair-tabs li a:hover{
+    border-bottom: 3px solid blue;
+    color: blue;
+}
+.container{
+    display: flex;
+    flex-flow: wrap;
+    padding: 0% 20% 0% 20%;
```
Comment (sobanakram) [inline sponsor-dashboard-style.css]: shorthand version `padding: 0% 20%;`

File: sponsor-dashboard-style.css [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+    margin-top: 30px;
+}
+.icon-style{
+    color: #C4183C;
+    padding: 13px 0px 0px 0px;
```
Comment (sobanakram) [inline sponsor-dashboard-style.css]: instead of adding zero padding, just use `padding-top` means specific

**Review by sobanakram — CHANGES_REQUESTED** [review]

**Comment by sobanakram** [comment]
@qaseem-aslam-brainx 
Points to note:
1. Indent code properly, you can use Vs Code to indent using keyboard shortcut.
2. avoid writing css classes with specific text which looks like can be changed, css should be independent of html.
3. use lowercase for colors and look into css variables.

**Comment by qaseem-aslam-brainx** [comment]
@sobanakram 
Is the 2nd point about class names like "fair-heading", "sponsor-list-table" etc. ?

File: css/sponsor-dashboard.css [hunk 12]
```diff
@@ -1,4 +1,10 @@
-#fair-heading{
+:root {
```
Comment (sobanakram) [inline css/sponsor-dashboard.css]: css variables should be in base style file which will be available in all views.

File: css/sponsor-dashboard.css [hunk 13]
```diff
@@ -90,17 +89,16 @@ table.sponsor-list-table td{
...
     border-radius: 10px;
     text-align: center;
     padding: 5px 20px 5px 20px;
 }
-.status-deactive{
-    background: #BECAD6;
-    border-radius: 10px;
-    text-align: center;
-    padding: 5px 20px 5px 20px;
+.active{
```
Comment (sobanakram) [inline css/sponsor-dashboard.css]: We should scope these with status as this is only concerned with status.
like `.status.active`

File: css/event-dashboard.css [hunk 14]
```diff
@@ -0,0 +1,153 @@
+:root {
+    --btn-background: #79d733;
+}
+.container{
```
Comment (sobanakram) [inline css/event-dashboard.css]: add this to common css file

File: css/nav-bar.css [hunk 15]
```diff
@@ -0,0 +1,71 @@
+* {
```
Comment (sobanakram) [inline css/nav-bar.css]: we can keep it as common css file and just `style.css` name works.

**Review by sobanakram — CHANGES_REQUESTED** [review]

**Review by sobanakram — APPROVED** [review]

**Pull request merged**